api/render.go

```diff
@@ -0,0 +1,63 @@
+//go:build ignore
```

Comment:
nit: add the copyright boilerplate (and to generate.go)

Reply:
Thanks! Ship it! :shipit:

Reply:
Thanks! But I can't merge. Mind doing it? And would you be ok with cutting a patch release with it?